LayoutTests/fast/scrolling/mac/wheel-event-deltas-are-not-filtered.html

```diff
@@ -53,6 +53,7 @@
                 }
             ];
 
+            await UIHelper.ensurePresentationUpdate();
             await UIHelper.mouseWheelSequence({ events });
```

Comment:
`mouseWheelSequence` already waits for a presentation update, via the one in `startMonitoringWheelEvents`.

Reply:
Waiting for a presentation update here does appear to fix the flakiness. Maybe the correct fix should be to wait for the presentation update before the call to `monitorWheelEvents`? Does that seem correct?

i.e.
`diff --git a/LayoutTests/resources/ui-helper.js b/LayoutTests/resources/ui-helper.js
index f37260d192f2..723b0dc656b8 100644
--- a/LayoutTests/resources/ui-helper.js
+++ b/LayoutTests/resources/ui-helper.js
@@ -72,8 +72,8 @@ window.UIHelper = class UIHelper {

     static async startMonitoringWheelEvents(...args)
     {
-        eventSender.monitorWheelEvents(args);
         await UIHelper.ensurePresentationUpdate();
+        eventSender.monitorWheelEvents(args);
     }`

Reply:
https://github.com/WebKit/WebKit/pull/12339, closing this one.